airsonic-main/src/main/java/org/airsonic/player/service/SearchService.java

```diff
@@ -33,24 +32,14 @@
 import java.util.List;
 
 /**
- * Performs Lucene-based searching and indexing.
+ * Performs Lucene-based searching.
  *
  * @author Sindre Mehus
  * @version $Id$
  * @see MediaScannerService
  */
 public interface SearchService {
 
-    void startIndexing();
-
-    void index(MediaFile mediaFile);
-
-    void index(Artist artist, MusicFolder musicFolder);
-
-    void index(Album album);
-
-    void stopIndexing();
-
     SearchResult search(SearchCriteria criteria, List<MusicFolder> musicFolders,

```

Comment:
If MediaScannerService references IndexManager instead of SearchService, there is no reason to expose Index related methods in the SearchService interface.
Removing these from the interface splits the search and indexing into a design.